### ISSUE TITLE
[Issue] : Add Single Product View Page and Product MRP 

### ISSUE THREAD

**Issue by codersalman** [saloni-15/Heel2Toe#88]
### Product MRP in Homepage Product Card and Single Product Page 

![image](https://user-images.githubusercontent.com/83803180/135765374-b97d58ac-9bfe-4688-911c-eb33738c2b65.png)

### Single Product Page

![image](https://user-images.githubusercontent.com/83803180/135765428-346ddfd4-92eb-4884-ac7c-2105323819bc.png)

### Are you a Hacktoberfest'21 contributor

- [X] Yes, I'am Hacktoberfest'21 contributor

**Comment by codersalman** [comment]
If it's a Valid Issue then Can you assign this to me ?

**Comment by saloni-15** [comment]
yes you can work, but there's already a product page, you can see here
https://saloni-15.github.io/Heel2Toe/product.html
you can redesign this accordingly

**Comment by codersalman** [comment]
Will Redesign it

**Comment by saloni-15** [comment]
hey @codersalman, are you working on this?

**Comment by codersalman** [comment]
Yes

**Comment by codersalman** [comment]
# New Product Card Layout 🎉

![Screenshot_20211005_183500.jpg](https://user-images.githubusercontent.com/83803180/136028526-15b4ee47-e37c-468d-8aa0-ff157c276591.jpg)


@saloni-15 

- [ ] New Product Card Layout
- [ ] Single Product Page Redesign

**Comment by saloni-15** [comment]
Looks good, but don't add this much of card-detail, just add one line (4-5 words)  card detail and write that MRP above the Buy Now button and instead of Rs can you use `₹` and let the buy now button be blue only, don't change it to red.

**Comment by codersalman** [comment]
http://github.codersalman.me/Heel2Toe/product.html

@saloni-15

**Comment by saloni-15** [comment]
The product page looks good, you can leave the cards as they were, remove the changes you have done in cards. I suggest you do a git pull upstream first, and make it even with the original repo, then again add your changes to the product files.

**Comment by codersalman** [comment]
Okay But it's done on my repo ![Screenshot_20211005_202644.jpg](https://user-images.githubusercontent.com/83803180/136048306-69e06299-c549-4661-8ae6-ccbced361b97.jpg)

Okay then I'm Pushing the Products Page Only in PR

**Comment by codersalman** [comment]
I've just opened PR checkout @saloni-15
